docs/clusters/gcp/install.md

```diff
@@ -31,14 +31,20 @@ zone: us-central1-a
 # instance type
 instance_type: n1-standard-2
 
-# GPU to attach to your instance (optional)
-accelerator_type: nvidia-tesla-t4
-
 # minimum number of instances
 min_instances: 1
 
 # maximum number of instances
 max_instances: 5
+
+# GPU to attach to your instance (optional)
+# accelerator_type: nvidia-tesla-t4
+
+# the name of the network in which to create your cluster
+# network: default
+
+# the name of the subnetwork in which to create your cluster
```

Comment:
Does GCP call this field subnetwork? If yes, we can inherit that name as well right?